### PR TITLE
Wrote tips for pycuda installation

### PR DESCRIPTION
**Pull request by unnonouno**
I made a tips page and wrote a tips for pycuda installation.

http://askubuntu.com/questions/643270/how-can-i-install-chainer-cuda-deps

related to #33

### CODE REVIEW

File: docs/source/tips.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Tips and FAQs
+=============
+
+Installation
+------------
+
+I cannot install pycuda
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to set ``PATH`` to CUDA bin path if you get an error below when you use ``pip install chainer-cuda-deps``::
```
Comment (beam2d) [inline docs/source/tips.rst]: `the` error below

**Comment by beam2d** [comment]
How about mentioning the typical path to the CUDA installation with NVIDIA's official installer for Linux: `/usr/local/cuda/bin`?

**Comment by unnonouno** [comment]
It's a good idea!

**Comment by unnonouno** [comment]
I fixed it. Please review it.

**Comment by beam2d** [comment]
LGTM!

**Pull request merged**